base-requirements.txt

```diff
@@ -6,7 +6,7 @@ docutils==0.12
 Markdown==3.3.4
 cmarkgfm==0.6.0
 Pillow==8.3.1
-psycopg2==2.8.6
+psycopg2-binary==2.8.6
```

Comment:
is this strictly necessary? could it be addressed in a separate hygiene PR?

Reply:
Just a note on why: I'm not fully certain how this affects deployment pipeline and don't want it to impact the ability to safely merge this.

Reply:
Moves this to https://github.com/python/pythondotorg/pull/2114